update.py

```diff
@@ -875,7 +875,7 @@ def create_features_page(features, build_options_by_define, vehicletype):
                 build_options = build_options_by_define[feature]
             except KeyError:
                 # mismatch between build_options.py and features.json
-                print("feature %s not in build_options.py" % feature)
+                # print("feature %s not in build_options.py" % feature)
```

Comment:
use a set() to not emit same warning lots of times

Reply:
Done.

@Hwurzburg could you test, please?